### PR TITLE
List builder pattern to site

### PR DESCRIPTION
**Pull request by mikearildbrown**

### CODE REVIEW

**Review by shey-v — DISMISSED** [review]

File: site/docs/patterns/list-builder.mdx [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+Only use a list builder when there are at least five items to choose from. If you have fewer items, use a single reorderable list.
+</ExampleContainer>
+
+## How to build
+
+### Anatomy
+
+The list builder has three functional areas:
+
+1. **Header**: Contains a list label (with optional item count), a button bar and a [filter](/salt/patterns/list-filtering).
```
Comment (lukeac123) [inline site/docs/patterns/list-builder.mdx:43]: the filter pattern doesn't currently exist ?

File: site/docs/patterns/list-builder.mdx [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+  <Diagram
+    src="/img/patterns/list-builder/stack-layout.png"
+    alt="A vertically aligned list builder."
+    contentPosition="top"
+    border
+  />
+
+<ExampleContainer type="positive" customPillText="Best practices">
+Use a minimum width of 8ems and ensure that the list builder spans the full width of the screen or its parent container.
```
Comment (lukeac123) [inline site/docs/patterns/list-builder.mdx:147]: NIT: why recommend in an em value instead of px / % ?

**Review by lukeac123 — APPROVED** [review]
LGTM - List Filtering pattern should be merged first, otherwise there is a dead link

File: site/docs/patterns/list-builder.mdx (hunk 2 above)
Comment (joshwooding) [reply site/docs/patterns/list-builder.mdx:147]: All min-widths are in ems

**Review by joshwooding — APPROVED** [review]

**Review by shey-v — APPROVED** [review]

**Pull request merged**